defaults/main.yml

```diff
@@ -14,6 +14,9 @@ rstudio_connect_sso_saml_metadata_content_b64_encoded: ""
 rstudio_encryption_key: ""
 rstudio_connect_encryption_key_location: "/var/lib/rstudio-connect/db/secret.key"
 
+rstudio_connect_user_id: -1
+rstudio_connect_group_id: -1
```

Comment:
I'm not sure if setting to `-1` is a good approach. Maybe a better idea would be to not define it at all?

Reply:
Decided to not define any default value for the variables